rp235x-hal-examples/README.md

```diff
@@ -102,18 +177,65 @@ Verifying Flash:    [==============================]  100%
 The device was rebooted to start the application.
 ```
 
-It is currently possible to build *some* of the examples in RISC-V mode. See
-[`riscv_examples.txt`](./riscv_examples.txt) for a list of the examples known to
-work. The missing ones probably rely on interrupts, or some other thing we
-haven't ported to work in RISC-V mode yet.
-
+At this point the development version is running on the RP2350 and the LED is blinking.
+When we look at `blinky` using `file` we see we have an RISC-V ELF file and using `ls`
+we see it is quite large, 2.4M:
```

Comment:
The size of an ELF is mostly irrelevant. What matters are the size of the output sections (aka segments) reported by the `rust-size` command that comes with `cargo-binutils`. So we could show that here.

Or we could defer to other reference material on this subject and not go into file size.

Reply:
I've dropped the file size, I did install cargo-binutils and used rust-size, interesting that text is slightly larger on ARM vs RISC-V at 8692 vs 7468, but bss is 4 vs 0.5M (0x80000) in release profile:
```
$ rust-size target/thumbv8m.main-none-eabihf/release/blinky
   text	   data	    bss	    dec	    hex	filename
   8692	      0	      4	   8696	   21f8	target/thumbv8m.main-none-eabihf/release/blinky

$ rust-size target/riscv32imac-unknown-none-elf/release/blinky
   text	   data	    bss	    dec	    hex	filename
   7468	      0	 524288	 531756	  81d2c	target/riscv32imac-unknown-none-elf/release/blinky
```